packages/opentelemetry-exporter-collector/test/browser/util.test.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as sinon from "sinon";
+import { sendWithXhr } from "../../src/platform/browser/util";
+import { ensureHeadersContain } from "../helper";
+
+describe('util - browser', () => {
+  let server: any;
+  const body = "";
+  const url = "";
+
+  let onSuccessStub: sinon.SinonStub;
+  let onErrorStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    onSuccessStub = sinon.stub();
+    onErrorStub = sinon.stub();
+    server = sinon.fakeServer.create();
+  });
+
+  afterEach(() => {
+    server.restore();
+    sinon.restore();
+  });
+
+  describe('when XMLHTTPRequest is used', () => {
+    describe('and Content-Type header is set', () => {
+      const explicitContentType = {
+        'Content-Type': 'application/json',
+      };
+      it('Request Headers should contain "Content-Type" header', done => {
```

Comment:
DRY:

if you add `beforeEach` block you can extract the same code from each tests and use it there

```
describe('......
let expectedHeaders

beforeEach(()=>{
  sendWithXhr(body, url, explicitContentType, onSuccessStub, onErrorStub);
  // ;charset=utf-8 is applied by sinon.fakeServer
  expectedHeaders = {
    'Content-Type': 'application/json;charset=utf-8',
  };
});
```

Reply:
i came close to it ! please look.